src/lib/asn1/ASN1OID.cpp

```diff
@@ -46,11 +46,11 @@ DLL_EXPORT OID ParseObjectID(const uint8_t * encodedOID, uint16_t encodedOIDLen)
         return kOID_NotSpecified;
     }
 
-    for (uint32_t i = 0; i < sOIDTableSize; i++)
+    for (const auto & oid : sOIDTable)
```

Comment:
The use of different names in the different loops in this file is a bit odd, but ok.